vfs/vfs_storage.go

```diff
@@ -312,6 +326,54 @@ func (d *Driver) MountVolume(req Request) (string, error) {
 	return volume.MountPoint, nil
 }
 
+func (d *Driver) bindMount(volume *Volume, opts map[string]string) (string, error) {
```

Comment:
No... This is not how it should be done.

These methods are deliberately private because there are duplicated logic everywhere to do mount. These interfaces are supposed to replace function call everywhere.

Code needs to be generalized and put in `util/volume.go`